_tuning-your-cluster/availability-and-recovery/snapshots/index.md

```diff
@@ -12,7 +12,7 @@ has_toc: false
 
 # Snapshots
 
-Snapshots are backups of a cluster's indexes and state. State includes cluster settings, node information, index metadata (mappings, settings, templates, etc.), and shard allocation.
+Snapshots are backups of a cluster's indexes and state. State includes cluster settings, node information, index metadata (mappings, settings, or templates), and shard allocation.
```

Comment:
"The state"?

Reply:
Normally it's referred to as just "state" (just like "disk").